rest-api/src/main/java/uk/gov/hmcts/ccd/definition/store/rest/endpoint/CaseDefinitionController.java

```diff
@@ -72,7 +72,17 @@ public CaseType dataCaseworkerIdAndJurisdictionIdCaseTypeGet(
         return caseRoleService.findByCaseTypeId(caseTypeId);
     }
 
-    @RequestMapping(value = "/data/jurisdictions/{jurisdiction_id}/case-type", method = RequestMethod.GET, produces = {"application/json"})
+    @RequestMapping(value = "/data/case-types-references", method = RequestMethod.GET, produces = {"application/json"})
```

Comment:
Can we make this endpoint more REST? I've done a bit of googling and this one is a nice solution to me:
https://stackoverflow.com/questions/28681099/rest-getting-specific-property-of-resources-in-collection-only
`/data/case-types?field=reference`
What do you think?

Reply:
Sure we can

Reply:
Actually had another thought and wouldn't it suggest we can also call `/data/case-types` which should not be possible due to perf implications?

Reply:
good point. We could add validation and return 400 bad request if they don't provide a field query param. Also we could return 400 for any field different than `reference`. i.e. for now we only support retrieving `reference`. 
Or, we could leave it as it is currently implemented. As you prefer

Reply:
Cool I will leave it than. Thanks 👍